src/screens/auth/SignIn.tsx

```
@@ -78,19 +72,19 @@ const SignIn: React.FC = () => {
-  // };
+  const init = async () => {
+    try {
+      setLoading(true);
+      const { sub } = await getTokenData(); // Assuming sub is the user identifier
+      const result = await getUser(sub);
+      const data = await getDocumentsList();
+      updateUserData(result?.data, data?.data);
+    } catch (error) {
+      setError(t("SIGNIN_ERROR_FETCHING_DATA"));
+      throw error;
+      setLoading(false);
```

Biome

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)